packages/utils/src/severity.ts

```diff
@@ -1,20 +1,28 @@
-import { Severity } from '@sentry/types';
+import { Severity, SeverityLevel } from '@sentry/types';
 
-import { SeverityLevel, SeverityLevels } from './enums';
+// TODO: Should `severityFromString` be deprecated?
+
+// Note: Ideally the `SeverityLevel` type would be derived from `validSeverityLevels`, but that would mean either
+//
+// a) moving `validSeverityLevels` to `@sentry/types`,
+// b) moving the`SeverityLevel` type here, or
+// c) importing `validSeverityLevels` from here into `@sentry/types`.
+//
+// Option A would make `@sentry/types` a runtime dependency of `@sentry/utils` (not good), and options B and C would
+// create a circular dependency between `@sentry/types` and `@sentry/utils` (also not good). So a TODO accompanying the
+// type, reminding anyone who changes it to change this list also, will have to do.
+
+export const validSeverityLevels = ['fatal', 'error', 'warning', 'log', 'info', 'debug', 'critical'];
 
-function isSupportedSeverity(level: string): level is Severity {
-  return SeverityLevels.indexOf(level as SeverityLevel) !== -1;
-}
 /**
- * Converts a string-based level into a {@link Severity}.
+ * Converts a string-based level into a member of the {@link Severity} enum.
  *
- * @param level string representation of Severity
+ * @param level String representation of Severity
  * @returns Severity
  */
 export function severityFromString(level: SeverityLevel | string): Severity {
-  if (level === 'warn') return Severity.Warning;
-  if (isSupportedSeverity(level)) {
-    return level;
-  }
-  return Severity.Log;
+  return (level === 'warn' ? 'warning' : validSeverityLevels.includes(level) ? level : 'log') as Severity;
 }
+
+// TODO Is this necessary?
+export type { SeverityLevel };
```

Comment:
Personally, I think this causes confusion where to import `SeverityLevel` from. So I'm in favour of exporting it from `@sentry/types` only.

Reply:
I am also in favor of not exporting this here - api surface area and all.